### ISSUE TITLE
Buy orders too high

### ISSUE THREAD

**Issue by SpookiPants** [Kenya-DK/quantframe-react#23]
I was previously using the previous stable release (1.0.4) and I moved to the most recent pre release(1.2.4).

Multiple times now the program has put in a buy order (which were fulfilled) which will be near impossible to make a profit from.

An example is Secondary Outburst bought at 340, when the sell orders are 320 just a few moments later, which the average price of the arcane at 300-320 the past 90 days.

This has also happened with Lato Vandal Barrel, as well as making incredibly small profit margins on almost every item vs the previous version giving profit margins that are to be expected.

Previously items would almost never be marked with the profit too low tag unless a seller was grossly undercutting everyone else. HALF of the items purchased in the pre release version are marked for too low profit.

---------------

I also noticed the SMA limit is setting the price lower than it has to. Example:

I buy an item for 100 and now im selling it at the lowest price of 115 (marked with sma limit) while the next lowest sell order is at 120. Why should the sma limit dictate that you make less plat? The sell order should always be at least at the lowest sell order.

Instead of an item being taken down from the market because the profit is too low, can the item just be listed at the lowest price that still will make the necessary profit? Sometimes the sellers that are selling the lowest price items are not online and a buyer will go to the next highest price.

**Comment by Kenya-DK** [comment]
Hmm, sounds weird.
Haven't seen it happen to anyone else. What are your settings?

The SMA is the avg for the last 7 days.

**Comment by SpookiPants** [comment]
Volume Threshold: 15
Range Threshold: 25
Average Price Cap: 1500
Max Total Price Cap: 100000
Price Shift Threshold: -1
Min SMA: 3
Trading Tax Cap: -1
Min Profit: 20, 25? i have since changed it

**Comment by Kenya-DK** [comment]
Ok..
The current data of the Secondary Outburst is.
```json
  {
    "url_name": "secondary_outburst",
    "item_id": "649322b07ec190215a693093",
    "order_type": "closed",
    "trading_tax": 8000,
    "volume": 27.714285714285715,
    "max_price": 341.42857142857144,
    "min_price": 297.85714285714283,
    "avg_price": 319.64285714285717,
    "moving_avg": 316.5,
    "median": 642.7142857142857,
    "range": 43.57142857142857,
    "week_price_shift": 4.5,
    "sub_type": {
      "rank": 5
    }
  }```
  But the price of the item is not stable when looks at the stasts, so meby thats is why.

**Comment by trowieuk** [comment]
Had the same thing happen for Secura Lecta, bought two for 100, sells for 30-40.

**Comment by SpookiPants** [comment]
Negative profit buy order, refreshed to check it was supposed to be deleted and it stuck around

![image_2024-08-28_231728988](https://github.com/user-attachments/assets/7a7f8782-ff55-4a5e-9f2f-dd83430b3451)

**Comment by Kenya-DK** [comment]
When i that from?

**Comment by SpookiPants** [comment]
The timestamp at the top shows the time.
It happened again on a different item so I looked into it and somebody was undercutting on the market.

**Comment by Kenya-DK** [comment]
Can y join the `discord

**Comment by Kenya-DK** [comment]
Fix in 1.2.5
